.env.example

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_DRUPAL_BASE_URL=https://example.ca
\ No newline at end of file
+NEXT_PUBLIC_DRUPAL_BASE_URL=http://example.ca
```

Comment:
@JoelJuniorManuelFombe, this should be `https://` because PWA's require SSL (that the `s` stands for)

Reply:
oh right, i will update here